### PR TITLE
updated the date part of the weather forecast so it returns day instead

### PR DESCRIPTION
**Pull request by gedefaws**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Based on the instructors suggestion, instead of returning the date as(MM-DD) weather /forecast now returns day as(Monday, Tuesday,...)  

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

using postman
- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: routes/weather.js [hunk 1]
```diff
@@ -147,7 +147,10 @@ router.get('/forecast/:zipcode', (request, res) => {
                     })
                     let weatherDescription = forecast[5].weather[0].description // Get the weather description for 12:00pm-3:00pm interval
                     let icon = forecast[5].weather[0].icon
-                    let weatherDate = (forecast[5].dt_txt).slice(5,10) //Get the date as only mm-dd format
+                    //let weatherDate = (forecast[5].dt_txt).slice(5,10) //Get the date as only mm-dd format
+                    var d = new Date(forecast[5].dt_txt);
+                    var days = ["Sunday","Monday","Tuesday","Wednesday","Thursday","Friday","Saturday"];
+                    let weatherDate = days[d.getDay()];
```
Comment (gtullio12) [inline routes/weather.js:153]: Won't d.getDay() return back a specific day like the 12th or the 4th? If so how can that work by indexing the days array?

Comment (gtullio12) [reply routes/weather.js:153]: Also can you run heroku local, and then hit the /forecast/zipcode endpoint and take a screenshot and send in the chat so I can see the result

Comment (gedefaws) [reply routes/weather.js:153]: d.getDay returns and int from 0-6 which is from sunday to monday.

Comment (gtullio12) [reply routes/weather.js:153]: gotcha

Comment (gtullio12) [reply routes/weather.js:153]: also next time i would avoid using the variable name 'd'. Should be more descrptive of what it is

Comment (gedefaws) [reply routes/weather.js]: I got how the getDay() method works from the internet and forgot to change the variable name. I got it fixed now.

**Pull request merged**